### PR TITLE
User story 17

### PR DESCRIPTION
**Pull request by KellyIB**
Fixed Tests for nav bar (moved merchant testing into merchant file, fixed small things)
Added test for user story 17
In Items Index, changed view to make Item images into links to the item show pages.

### CODE REVIEW

File: app/views/items/index.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <section class = "grid-item" id= 'item-<%=item.id%>'>
       <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
       <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-      <img src= <%= item.image %>>
+      <%= link_to image_tag(item.image), "items/#{item.id}" %>
```
Comment (adumortier) [inline app/views/items/index.html.erb:12]: nice link! :)

**Review by cgaddis36 — COMMENTED** [review]
Good work on finding the correct syntax for finding an image and clicking it!

File: spec/features/nav_bar/merchant_navigation_spec.rb [hunk 2]
```diff
@@ -12,13 +12,21 @@
...
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@merchant_user)
-      
     end
 
-    it "sees the regular user links and the merchant dashboard link" do 
+    it "I can see a link to my profile pages" do
+      visit '/merchants'
+
+      within 'nav' do
+        expect(page).to have_content("Profile")
```
Comment (cgaddis36) [inline spec/features/nav_bar/merchant_navigation_spec.rb:22]: Not positive we need this tested twice

**Review by cgaddis36 — APPROVED** [review]

File: spec/features/nav_bar/merchant_navigation_spec.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'Site Navigation' do
-  describe 'As a User' do
+  describe 'As a merchant' do
```
Comment (adumortier) [inline spec/features/nav_bar/merchant_navigation_spec.rb:4]: nice catch!

**Review by adumortier — APPROVED** [review]

**Pull request merged**